data/pages-content/volunteers.md

```diff
@@ -0,0 +1,89 @@
+---
+
+EuroPython is organized and run by volunteers from the Python community, but we're only a few and need your help to make the show run as smoothly as possible, so we'd appreciate more help from the EuroPython attendees, if possible.
+
+This page explains some of the tasks we have available for you.
+
+Being a volunteer means to give your time for free, without expecting anything back, just for the pleasure of being a part of this amazing conference.
+
+In return, we will have a small number of perks for you, some small gifts to say "thank you" for your time and for your help.
+
+
+# Things to remember
+  - To enter the conference you need a ticket
+  - Yo can sign up to any tasks on our Volunteer App before you have bought a ticket
+  - You will not earn a ticket for being a volunteer
+
+# How to sign up 
+  - Acknowledge our [EuroPython 2022 Data Privacy Policy](https://docs.google.com/forms/d/e/1FAIpQLSdNTLsFn-YKD4l6icY4NEhmVG5JRyawm-QcvZUQ9HSylEXldw/viewform). This is needed, since many of you will have to handle private data as part of e.g. helping at the registration desk.
+  - Assign yourself to available shifts on the Operations spreadsheet. (You will get the link after signing the Data Privacy Policy.) 
+  - Sign up to the Europython Volunteers discord server. (The link is in our Operations Spreadsheet)
+  - Please say hello and introduce yourself in #onsite-volunteers channel. Thanks <3
+
+# Perks for volunteers
+
+  - Get an awesome EuroPython Volunteer T-Shirt that you can keep and show off to your friends and a mention on the website to stay there forever :-) (1 slot activity)
+  - A special group photo with all volunteers (1 slot activity; time will be announced on the #onsite-volunteers)
+
+
+# Summary of tasks
+
+### Registration Desk
+  - Hand out registration badges
+  - Check student IDs
+  - Register On-Desk tickets and day passes
+  - Answer general questions
+  - This job involves a lot of talking
+
+### Session Chairing
+  - Help speakers in preparing their notebooks for the talks
+  - Introduce the speakers to the audience
+  - Signal time left to the speakers
+  - Make sure talks are finished on time
+  - Moderate Q&A session and help with passing around microphones and introduce remote attendees.
+  - This job involves speaking in front of people, moderation, and walking with a microphone
+
+### T-Shirt Handout
+
+  - Hand out the conference T-shirt
+  - This job involves standing and lifting light items
+
+### Set Up/Tear Down
+
+  - Helping hands are needed to set up the stage before the conference starts (Monday morning)
+  - Also on the last day (Friday afternoon for the main conference, Sunday for the sprints) to clear up the venue afterwards
+  - This job involves walking and carrying items
+
+## Ad-hoc Activities
+
+  - There are always some tasks which we simply cannot plan beforehand:
+    * moving chairs, tables, or poster walls,
+    * guiding groups between the two venues,
```

Comment:
We thankfully don't have two venues this time.

Reply:
Do you think we have to keep this section? I was not 100% sure.